clicache/integration-test2/SerializationTests.cs

```diff
@@ -0,0 +1,131 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.IO;
+using Xunit;
+using PdxTests;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace Apache.Geode.Client.IntegrationTests
+{
+
+    [Trait("Category", "Integration")]
+    public class SerializationTests : IDisposable
+    {
+        private GeodeServer GeodeServer;
+        private CacheXml CacheXml;
+        private Cache Cache;
+
+        public SerializationTests()
+        {
+            GeodeServer = new GeodeServer();
+            CacheXml = new CacheXml(new FileInfo("cache.xml"), GeodeServer);
+
+            var cacheFactory = new CacheFactory();
+            Cache = cacheFactory.Create();
+            Cache.InitializeDeclarativeCache(CacheXml.File.FullName);
+        }
+
+        public void Dispose()
+        {
+            try
+            {
+                if (null != Cache)
```

Comment:
Probably unnecessary since if `Cache` is null you have bigger problems, but depends where you value failure notifications.  I generally prefer throwing the null pointer error since you initialize the cache before each test.

Reply:
I am hoping to avoid the test framework giving me extra exceptions to track down should the creation of cache fail and then a I get a null reference trying to clean it up.

Reply:
Cool, was thinking it would tell you what line during the throw, but also if you use cache in every test and it was null the exception would be thrown before this dispose anyway.  Carry on